boards.txt

```diff
@@ -809,36 +936,50 @@ adafruit_crickit_m0.menu.debug.off=Off
 adafruit_crickit_m0.menu.debug.on=On
 adafruit_crickit_m0.menu.debug.on.build.flags.debug=-g
 
+
+# -----------------------------------
 # Adafruit Metro M4 (SAMD51)
-# ------------------------------
+# -----------------------------------
 adafruit_metro_m4.name=Adafruit Metro M4 (SAMD51)
+
+# VID/PID for Bootloader, Arduino & CircuitPython
 adafruit_metro_m4.vid.0=0x239A
 adafruit_metro_m4.pid.0=0x8020
 adafruit_metro_m4.vid.1=0x239A
 adafruit_metro_m4.pid.1=0x0020
-adafruit_metro_m4.upload.tool=bossac18
+adafruit_metro_m4.vid.2=0x239A
+adafruit_metro_m4.pid.2=0x8021
+adafruit_metro_m4.vid.3=0x239A
+adafruit_metro_m4.pid.3=0x0021
+
+# Upload
+adafruit_metro_m4.upload.tool=bossac19
 adafruit_metro_m4.upload.protocol=sam-ba
 adafruit_metro_m4.upload.maximum_size=507904
 adafruit_metro_m4.upload.offset=0x4000
 adafruit_metro_m4.upload.use_1200bps_touch=true
 adafruit_metro_m4.upload.wait_for_upload_port=true
 adafruit_metro_m4.upload.native_usb=true
+
+# Build
 adafruit_metro_m4.build.mcu=cortex-m4
-#adafruit_metro_m4.build.f_cpu=120000000L
-adafruit_metro_m4.build.usb_product="Adafruit Metro M4"
-adafruit_metro_m4.build.usb_manufacturer="Adafruit LLC"
+adafruit_metro_m4.build.f_cpu=120000000L
+adafruit_metro_m4.build.usb_product="Metro M4"
```

Comment:
Adafruit should be dropped from `usb_product` otherwise Adafruit could be shown twice in client such as lsusb since it is common for client to display `vendor + product` string together. Let me know if it is ok, or still prefer to keep it.

```
Bus 003 Device 012: ID 239a:801b Adafruit Feather M0 Express
```

Reply:
OK cool!